fix: update the node azure install docs (#14425)
newrelic/docs-website: src/content/docs/apm/agents/nodejs-agent/hosting-services/nodejs-agent-microsoft-azure.mdx

* Install the Azure SDK for Node.js.
* Ensure Git is installed.

<Callout variant="important">

>   The Node.js agent can not instrument Azure functions, for that you will need to use the [Azure Functions monitoring integration](docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-functions-monitoring-integration/)

Reviewer: ```suggestion
  The Node.js agent cannot deliver telemetry for Azure Functions. Instead, you will need to use the [Azure Functions monitoring integration](docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-functions-monitoring-integration/)
```
